.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: node_js
+
 node_js:
 - 10
+- 12
+- 14
 - lts/*
```

Comment:
The current LTS release is 14 right now, so these two are identical.

IMO, 12 is also unnecessary. If it passes in node 10, then it's not using any newer APIs, and if it passes in LTS and stable, then it's not using any deprecated APIs. I find it unlikely that a breaking change would be rolled out to node 12 without but not 10 or 14 or 15.

Reply:
Ok I'll revert these changes